Makefile

```diff
@@ -8,6 +8,18 @@ TAR := $(shell if which gnutar 1>/dev/null 2> /dev/null; then echo gnutar; else
 
 default: wannier post
 
+PREFIX ?= /usr
+
+install:
```

Comment:
Question (I don't know the answer): in other projects, does `install` depend on make, or it just installs what has been compiled? (Possibly it's ok as you did, just to double check)

Reply:
I checked the gnu make project, invoking `make install` will compile & install. So I add `default` target as a dependency of `install`.